sdk/src/wallet/core/mod.rs

```diff
@@ -410,32 +369,55 @@ where
         self.inner.emit(wallet_event).await
     }
 
-    pub async fn data(&self) -> tokio::sync::RwLockReadGuard<'_, WalletData> {
-        self.data.read().await
+    /// Get the wallet address config.
+    pub async fn address(&self) -> Bech32Address {
+        self.address.read().await.clone()
     }
 
-    pub(crate) async fn data_mut(&self) -> tokio::sync::RwLockWriteGuard<'_, WalletData> {
-        self.data.write().await
+    pub(crate) async fn address_mut(&self) -> tokio::sync::RwLockWriteGuard<'_, Bech32Address> {
+        self.address.write().await
     }
 
-    #[cfg(feature = "storage")]
-    pub(crate) fn storage_manager(&self) -> &StorageManager {
-        &self.storage_manager
+    /// Get the wallet's configured Bech32 HRP.
+    pub async fn bech32_hrp(&self) -> Hrp {
+        self.address.read().await.hrp
+    }
+
+    /// Get the wallet's configured bip path.
```

Comment:
```suggestion
    /// Get the wallet's bip path.
```

Reply:
Seems like it was committed but not auto-closed.

Reply:
It was not committed

Reply:
ooh, I see